index.html

```diff
@@ -0,0 +1,270 @@
+            Back End Dev
+          </li>
+
+          <li>
+            <img src="images/Counter.png" alt="" />
+            2015
+          </li>
+        </ul>
+        <!-- </div> -->
+
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+
+        <div class="list-container-2">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <button type="button" class="big-button">See project</button>
+      </div>
+      <!---about my self-->
+      <section class="workd-card about-myself">
+        <h1>About Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h4 id="text-color">LET’S CONNECT</h4>
+
+        <ul>
+          <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+          <li><a href="#"><i class="fab fa-linkedin"></i></a></li>
+          <li><a href="#"><i class="fab fa-medium"></i></a></li>
+          <li><a href="#"><i class="fab fa-github"></i></a></li>
+          <li><a href="#"><i class="fab fa-angellist"></i></a></li>
+        
+        </ul>
+
+        <button type="button" class="big-button below-button">
+          Get my resume
+        </button>
+      </div>
+      <!---about my self end-->
+
+      <div class="frames">
+        <div class="down-arrow space">
+          <h4 class="alignment">
+            <ul>
+              <li>languages</li>
+            </ul>
+           
+            
+            <i class="fa fa-arrow-down"></i>
+          </h4>
+        </div>
+
+        <img src="images/Frame9.png" alt="" />
+        <img src="images/Frame10.png" alt="" />
+        <img src="images/Frame11.png" alt="" />
+
+        <img src="images/Vector.png" alt="" />
+        <h4 class="down-arrow">
+          <ul><li>Frameworks</li></ul>
+          
+
+          <i class="fa fa-arrow-right"></i>
+        </h4>
+        <img src="images/SeparatorBottom.png" alt="" />
+
+        <h4 class="down-arrow">
+          <ul>
+            <li>Skills</li>
+          </ul>
+          
+          <i class="fa fa-arrow-right"></i>
+        </h4>
+        <img src="images/SeparatorBottom.png" alt="" />
+      </div>
+    </section>
+    <!-- Contact section -->
+ <section class="contact">
+  <div class="container-contact">
+    <div class="container-aligment">
+  <h2>Contact me</h2>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+</div>
+<div class="form">
+  <form action="https://formspree.io/f/xgeqkbbo" method="post">
+    <input type="text" placeholder="Alexandra Loremipsum" maxlength="30" required>
```

Comment:
- [ ] Kindly use your second name instead of "Loremipsum" for a better user interface. 👍🏽